agent/api/task/taskvolume.go

```diff
@@ -78,6 +79,9 @@ func (tv *TaskVolume) UnmarshalJSON(b []byte) error {
 		return tv.unmarshalFSxWindowsFileServerVolume(intermediate["fsxWindowsFileServerVolumeConfiguration"])
 	case apiresource.EBSTaskAttach:
 		return tv.unmarshalEBSVolume(intermediate["ebsVolumeConfiguration"])
+	case AttachmentType:
+		seelog.Warn("Obtaining the volume configuration from task attachments.")
```

Comment:
is this an expected path?  if so can this be Debug or Info level?

Reply:
Trying to understand, why is this no-op for Attachment type? will this be updated in future PR?

Reply:
Good question, so we want this to be a noop for volume of type `attachment` since it'll end being removed and replaced by the newly created volume configuration that we make when handling the task attachments. 

We originally thought to try to have a special edge case for volumes defined in the list of task attachments here in `taskvolume.go` but that would mean we'd have to change around some of the existing edge cases and can get messy.